Makefile

```diff
@@ -1,5 +1,5 @@
 create:
-	rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
+	printf "Y\n" | rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
```

Comment:
Please add the comment what does the `Y` do here @namtx, you can check how other addon did that.